Clicker/Views/Settings/SettingsLogOutButtonCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SettingsLogOutButtonCell.swift
+//  Clicker
+//
+//  Created by Lucy Xu on 3/23/19.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol LogoutButtonCellDelegate: class {
+    func logOutButtonCellWasTapped()
+}
+
+class SettingsLogOutButtonCell: UICollectionViewCell {
+    
+    // MARK: Data vars
+    weak var delegate: LogoutButtonCellDelegate?
+    
+    // MARK: Views and VC's
```

Comment:
idk if there are view controllers here so you should just comment it as `Views`

Reply:
Addressed in most recent pull request!